Packs/Darktrace/ReleaseNotes/3_0_13.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Darktrace Model Breaches
+
+UV support for acknowledging and unacknowledging model alerts.
```

Comment:
Release notes must start with one of the approved prefixes. Please adjust your note as per the [release notes guidelines](https://xsoar.pan.dev/docs/documentation/release-notes#:~:text=Release%20notes%20must,a%20new%20playbook%27).

Reply:
@YairGlik Added prefix as you requested.